src/dummy_writer.jl

```diff
@@ -0,0 +1,19 @@
+"""The `DummyWriter` does nothing. It can be used to temporarily disable output of a
+scheduled diagnostic. Mostly useful for testing and debugging ClimaDiagnostics.
+"""
+struct DummyWriter <: AbstractWriter end
```

Comment:
We could define this in the test suite if it's only used for tests, but I'm fine with leaving it here if it helps with debugging

Reply:
It is used only the tests, but I personally find it useful while debugging. It is a very simple way to disable output while keeping compute active.